config/allowed-redirects.txt

```diff
@@ -20,3 +20,15 @@ https://www.sublimelinter.com/	http://www.sublimelinter.com/en/master/
 
   This site seems to redirect to a honeypot
 https://microformats.org/	https://qltuh.check-tl-ver-{(.*)}/space-robot/{(.*)}
+
+  login redirections
+https://testingbot.com/members{(.*)}	https://testingbot.com/users/sign_in
```

Comment:
In `check-external-link.ts`, you'll see I actually have generic logic for redirections to login. Currently it just detects `/\/(login|signin)\b/`, but you can definitely add `sign_in` to the regex.